docs/data/Allas/introduction.md

```diff
@@ -0,0 +1,105 @@
+>>>>>>> ea5136b17c371be5e0e176abb28a30738bd880d5
+
+2. WWW access to Allas is provided by the web interface of cPouta cloud environment [https://pouta.csc.fi](https://pouta.csc.fi). To access Allas with a browser you naturally do not have to install anything special on your computer, so this is by far the simplest way to access Allas. On the other hand, the browser user interface has a bunch of limitations compared to other access flavors, most notable are less performance and upload/download only a single file at the time. The instructions for accessing and using Allas with a browser can be found from [OpenStack Horizon web interface](./using_allas/web_client.md){:target="_blank"}.
+
+3. To access Allas with a GUI client you have to install a suitable GUI client in to your server/laptop. The client needs to be capable to use Swift or S3 access protocol. Instructions to use one such client can be found from [Accessing Allas with Windows and Mac](./accessing_allas.md#accessing-allas-with-windows-and-mac){:target="_blank"}.
+
+4. To access Allas with command-line commands you need to install Swift or S3 protocol supporting client software in to your server/laptop. This is the most flexible way to access Allas, but it requires more effort from you than other access flavors. Instructions to use one such client can be found from [Accessing Allas with Linux](./accessing_allas.md#accessing-allas-with-linux){:target="_blank"} and [Protocols](./accessing_allas.md#protocols){:target="_blank"}.
+
+In addition to these access flavors you can also see how to use Allas based on [Common Use Cases](./using_allas/common_use_cases.md){:target="_blank"}.
+
+
+&nbsp;
+
+
+## Billing and Quotas
+
+Allas usage is founded on project based storage quotas. The default quota for a new project is 1 TB, but that can be increased if needed. Allas is the preferred storage site for any large datasets in CSC environment, so you should not hesitate to request larger quota for Allas, if you need to work with larger data sets. 
+
+ All the project members have equal access rights to the storage area that has been granted for the project. In practice, this means that if one project member uploads data to Allas, all the other project members can read, edit and also delete the data. Allas itself does not store any information about who has uploaded the data to Allas.
+
+The default quotas for every project are:
+
+| Resource | Limit |
+| :-------- |:------- |
+| Storage amount | 1 TiB |
```

Comment:
Quotas have not been updated. Correct quotas are here
https://wiki.eduuni.fi/display/CSCSR/Agenda+SR-JR+COMP+2019-09-05